taskcluster/tc-build-utils.sh

```diff
@@ -337,6 +410,116 @@ do_nuget_build()
   nuget pack nupkg/deepspeech.nuspec
 }
 
+do_nuget_repackage()
```

Comment:
so clearly, i think you can just remove home.tar.xz from the payload, as well as pyenv.tar.gz